### DISCUSSION TITLE
Option to Combine Live Photo Image and Video into One File for Sharing

### DISCUSSION THREAD

**Discussion by Jishnuraj9** [ente-io/ente#4516] [Enhancements]
When sharing Live Photos from the Ente app with people who don’t use Ente (e.g., via AirDrop), two separate files are sent: one for the photo and one for the video. This can be confusing for recipients.

In the default Photos app, Live Photos are shared as a single file that combines both the image and video, making it much easier for others to view.

It would be great if Ente could have an option to either combine the image and video into one file when sharing, or provide a toggle in the app settings so users can choose whether to send both or just the photo.

**Comment by jpsalo** [comment]
Agree! I've always considered Live Photos as photos and that should be the default sharing setting

**Comment by edbourque0** [comment]
I agree with this request. This is the main reason why I'm not using Ente as my main photos app.
